src/scancode/api.py

```diff
@@ -270,6 +271,7 @@ def _licenses_data_from_match(
         matched_rule['is_license_reference'] = match.rule.is_license_reference
         matched_rule['is_license_tag'] = match.rule.is_license_tag
         matched_rule['is_license_intro'] = match.rule.is_license_intro
+        matched_rule['has_unknown'] = match.rule.has_unknown
```

Comment:
I think it should be present only once, and probably outside `matched_rule` since we are reporting this as a property of the match, and not rule.

Reply:
@AyanSinhaMahapatra It is fine to keep for now at the rule level. This is the way for other attributes.